cypress/e2e/8-motions.js

```diff
@@ -117,6 +118,66 @@ describe('User can create motions via UAC', () => {
     cy.checkMotion();
   });
 
+  it('Can claim tokens', () => {
+    cy.login();
+    cy.visit(`/colony/${Cypress.config().colony.name}`);
+    // // activate tokens
+    cy.getBySel('tokenActivationButton', { timeout: 120000 }).click();
+    cy.getBySel('inputMaxButton').click();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.getBySel('tokenActivationConfirm').click().wait(10000);
```

Comment:
I think that this should be checking that the token activation has worked. As in my runs, it passes even without the tokens being activated.